code/game/mecha/mecha.dm

```diff
@@ -1183,7 +1183,7 @@
 		dir = dir_in
 		log_message("[mmi_as_oc] moved in as pilot.")
 		if(!hasInternalDamage())
-			to_chat(occupant, sound(nominalsound, volume=50))
```

Comment:
Who the heck did this.
![image](https://user-images.githubusercontent.com/25063394/167410811-da6e57b9-4076-4b93-9bc8-8379e79706b5.png)
TIGERRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRR

Reply:
Slightly wrong tiger. The https://github.com/ParadiseSS13/Paradise/pull/4107 is the one you're looking for
![20220509-200727-firefox](https://user-images.githubusercontent.com/7831163/167489105-053f5417-82d8-4d5d-a683-a988dc8b46d1.png)

Reply:
Credit for the dive